frontend/resourceadm/pages/AboutResourcePage/AboutResourcePage.tsx

```diff
@@ -189,7 +182,7 @@ export const AboutResourcePage = ({
    *
    * @param s the selected string
    */
-  const onChangeResourceType = (s: string) => {
+  const handleChangeResourceType = (s: string) => {
```

Comment:
What does "s" mean here? s = string? Should we try to come up with a more descriptive name? 🤔